test/Integration/ByteFormatterTest.php

```diff
@@ -77,7 +77,11 @@ public function provideDecimalIntegers()
     /** @dataProvider providePrecisionIntegers */
     public function testPrecision($integer, $formatted)
     {
-        $this->assertSame($formatted, $this->formatter->setPrecision(2)->format($integer));
+        $withArgument = $this->formatter->setPrecision(5)->format($integer, 2);
+        $withMethod = $this->formatter->setPrecision(2)->format($integer);
+
+        $this->assertSame($formatted, $withMethod);
+        $this->assertSame($withMethod, $withArgument);
```

Comment:
The way this is written is a bit weird! It's not really important to compare `$withMethod` and `$withArgument` when what we really want to know is that `$withArgument` still matches `$formatted`. Since we already verified that `$withMethod` and `$formatted` are the same in the previous line it makes no difference, as long as these assertions continue to appear in this order, but by fixing this comparison it means the assertions read correctly in isolation.

Moreover, we eliminate the need to store the results in variables, this eliminating `$withArgument` and `$withMethod` variables. If you'd like to make this change that would be helpful, otherwise I can merge as-is and make the change myself later.